include/swift/AST/DiagnosticsParse.def

```diff
@@ -692,12 +692,14 @@ ERROR(multiple_parameter_ellipsis,none,
       "only a single variadic parameter '...' is permitted", ())
 ERROR(parameter_vararg_default,none,
       "variadic parameter cannot have a default value", ())
-ERROR(parameter_inout_var_let,none,
+ERROR(parameter_inout_var_let_repeated,none,
       "parameter may not have multiple 'inout', 'var', or 'let' specifiers",
       ())
+ERROR(parameter_let_as_attr,none,
+	  "'let' before a parameter type is not allowed, place it before the parameter name instead", ())
+ERROR(parameter_var_as_attr,none,
+"'var' as a parameter attribute is not allowed, consider creating a shadow copy in the function body or using 'inout' for pass-by-reference", ())
```

Comment:
Please indent the "'var...  line to match the placement of the "(parameter_var_as_attr" on the previous line.  Similarly, please de-indent the "'let' before ... " line a few lines above.